routes/web.php

```diff
@@ -14,15 +15,22 @@
 | contains the "web" middleware group. Now create something great!
 |
 */
+
+Route::controller(ShowController::class)->group(function() {
```

Comment:
- ShowControllerが差分に入ってない(commitできてる,,,?
- 名前これに変えた方がいいと思う

```suggestion
Route::controller(ArticleController::class)->group(function() {
```

Reply:
修正済。